app/views/whats_new/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<% content_for :page_title, t('whats_new.title') %>
```

Comment:
I don't think it's necessary to use a locale file for this. In Whitehall this has been set-up as a makeshift CMS for a content designer to edit (so they work with YAML). Given we're likely only go to do about 5-6 edits to this page we may as well remove the need for the extra abstraction and just have the content directly in the view.